test/fuzz/test_runner.py

```diff
@@ -69,7 +69,8 @@ def main():
     )
     parser.add_argument(
         '--m_dir',
-        help='Merge inputs from this directory into the corpus_dir.',
+        action="append",
+        help="Merge inputs from these directories into the corpus_dir.",
```

Comment:
It could perhaps be clarified that the `--m_dir=` parameter can be repeated with multiple directories. My first instinct was to use it as the last parameter and try to provide multiple directories with the help text being “from these directories”. Works great when repeated, though.